lib/topological_inventory/orchestrator/metric_scaler.rb

```diff
@@ -1,46 +1,55 @@
 require "manageiq-loggers"
+require "topological_inventory/orchestrator/object_manager"
+require "topological_inventory/orchestrator/metric_scaler/watcher"
+require "topological_inventory/orchestrator/metric_scaler/prometheus_watcher"
 
 module TopologicalInventory
   module Orchestrator
     class MetricScaler
       attr_reader :logger
 
-      def initialize(logger = nil)
-        require "topological_inventory/orchestrator/metric_scaler/watcher"
+      def initialize(prometheus_hostname, logger = nil)
         @logger = logger || ManageIQ::Loggers::CloudWatch.new
+        @prometheus_hostname = prometheus_hostname
         @cache  = {}
       end
 
       def run
+        logger.info("#{self.class.name}##{__method__} Starting...")
```

Comment:
With this moved here we only see the log message when the pod starts correct?  Which may be fine, but the original behavior did this every 10 seconds.  

I guess I'm not sure if we need to see this every 10 seconds, so potentially fine. @slemrmartin do you have any thoughts?

Reply:
as you said we don't need this info message each 10 seconds, it doesn't give us too valuable information so this is fine to me